### ISSUE TITLE
Redesign of alt-text indicator

### ISSUE THREAD

**Issue by Bossett** [bluesky-social/social-app#3712]
**Is your feature request related to a problem? Please describe.**

![image](https://github.com/bluesky-social/social-app/assets/114112113/a8c2fa8e-fe79-4c1d-b65d-2a24b8c21afe)

The alt-text indicator often obscures part of the image.

**Describe the solution you'd like**

Move the alt indicator below the image, and include the alt text alongside:

![image](https://github.com/bluesky-social/social-app/assets/114112113/4633f374-45d4-4a3c-8902-e847fddc005c)

With ellipses to constrain text if that's a concern:

![image](https://github.com/bluesky-social/social-app/assets/114112113/fa54c945-2afa-4f56-bdcc-5da9075b0012)

**Additional context**

* Care should be taken so that only one batch of alt-text is read by screen readers/accessibility tools, on encountering the image.
* This will raise the profile of alt as it'll draw the eye to the 'secret' text
* This will remove one of the 'punishments' put on those that use alt (i.e. their images are not obscured)

**Comment by cosmostellar** [comment]
It shows the alt message when you click and view the image. I personally think it's much cleaner this way. I wonder what you think.

<img width="894" alt="image" src="https://github.com/bluesky-social/social-app/assets/160427240/94b068c0-2d9a-45f6-8f49-257af9d29260">

**Comment by Bossett** [comment]
Can leave the on-click behaviour - but looking for a solution to move the ALT indicator (that currently obscures the image) without it feeling 'unbalanced' by having a mass of black space to its right.

Having it displayed as a caption may not be the best way to do it - but something else in that space I think is important to make it not look 'wrong'

**Comment by cosmostellar** [comment]
Yeah. That's a good point. I also don't like how it covers the entire screen, while I can still read the post on X (or Twitter).

<img width="1051" alt="image" src="https://github.com/bluesky-social/social-app/assets/160427240/ea258a42-22b6-4f08-b690-14eaa3b1c299">

\
BlueSky puts the alt message in the image tag, but it can definitely be improved.

<img width="498" alt="image" src="https://github.com/bluesky-social/social-app/assets/160427240/4130e045-8e6d-4d28-b6df-bad9cf374c51">
<img width="439" alt="image" src="https://github.com/bluesky-social/social-app/assets/160427240/00157c50-5500-48d3-8152-145e896f4c54">

**Comment by cosmostellar** [comment]
On X (Twitter), you still need to hover on the alt block to view the message though. I think it could display the alt message below the image when you click or hover on the block.

<img width="325" alt="image" src="https://github.com/bluesky-social/social-app/assets/160427240/774c7d17-3244-4cb8-8f16-ef62b0daa202">
